### PR TITLE
Add multi-arch support document for arm64 support

### PR DESCRIPTION
**Pull request by boknowswiki**
https://github.com/longhorn/longhorn/issues/6

Signed-off-by: Bo Tao <bo.tao@rancher.com>

### CODE REVIEW

File: content/docs/1.1.0/what-is-longhorn.md [hunk 1]
```diff
@@ -4,10 +4,13 @@ weight: 1
 ---
 Longhorn is a lightweight, reliable and easy-to-use distributed block storage system for Kubernetes.
 
+Longhorn supports AMD64 and ARM64 both architectures.
+
 Longhorn is free, open source software. Originally developed by Rancher Labs, it is now being developed as a sandbox project of the Cloud Native Computing Foundation.
 
 With Longhorn, you can:
 
+- Deploy Longhorn in AMD64 or ARM64 cluster environments with the same multi-arch images.
```
Comment (yasker) [inline content/docs/1.1.0/what-is-longhorn.md:13]: Don't need.

File: content/docs/1.1.0/what-is-longhorn.md [hunk 2]
```diff
@@ -44,4 +47,4 @@ Common front-ends include a Linux kernel device (mapped under /dev/longhorn) and
 
 ### Specify Schedules for Recurring Snapshot and Backup Operations
 
-Specify the frequency of these operations (hourly, daily, weekly, monthly, and yearly), the exact time at which these operations are performed (e.g., 3:00am every Sunday), and how many recurring snapshots and backup sets are kept.
\ No newline at end of file
+Specify the frequency of these operations (hourly, daily, weekly, monthly, and yearly), the exact time at which these operations are performed (e.g., 3:00am every Sunday), and how many recurring snapshots and backup sets are kept.
```
Comment (yasker) [inline content/docs/1.1.0/what-is-longhorn.md:50]: ?

**Review by yasker — CHANGES_REQUESTED** [review]

File: content/docs/1.1.0/what-is-longhorn.md (hunk 2 above)
Comment (boknowswiki) [reply content/docs/1.1.0/what-is-longhorn.md:50]: I think this is caused my vim format. When I do 'git diff' it shows me
```
-Specify the frequency of these operations (hourly, daily, weekly, monthly, and yearly), the exact time at which these operations are performed (e.g., 3:00am every Sunday), and how many recurring snapshots and backup sets are kept.
\ No newline at end of file
+Specify the frequency of these operations (hourly, daily, weekly, monthly, and yearly), the exact time at which these operations are performed (e.g., 3:00am every Sunday), and how many recurring snapshots and backup sets are kept.
```

File: content/docs/1.1.0/what-is-longhorn.md [hunk 3]
```diff
@@ -4,10 +4,13 @@ weight: 1
 ---
 Longhorn is a lightweight, reliable and easy-to-use distributed block storage system for Kubernetes.
 
+Longhorn supports AMD64 and ARM64 both architectures.
```
[the remark this answers is not in the document]
Comment (boknowswiki) [reply content/docs/1.1.0/what-is-longhorn.md:7]: Will fix it. Thanks.

File: content/docs/1.1.0/what-is-longhorn.md (hunk 1 above)
Comment (boknowswiki) [reply content/docs/1.1.0/what-is-longhorn.md:13]: Will fix it thanks.

**Review by yasker — APPROVED** [review]

**Pull request merged**